pkg/local_object_storage/engine/select.go

```diff
@@ -65,6 +65,44 @@ func (e *StorageEngine) Select(prm *SelectPrm) (*SelectRes, error) {
 	}, nil
 }
 
+// List returns `limit` available physically storage object addresses in engine.
+// If limit is zero, then returns all available object addresses.
+func (e *StorageEngine) List(limit uint64) (*SelectRes, error) {
+	addrList := make([]*object.Address, 0)
```

Comment:
Correct, but we also can allocate some approximate capacity (`limit`).

Reply:
That's right, thank you. Fixed.